app/src/main/java/ru/vssemikoz/newsfeed/utils/TypeConverters/DateConverter.java

```diff
@@ -7,7 +7,8 @@
 import java.util.Date;
 
 public class DateConverter {
-    final static String dateFormat = "yyyy-MM-dd'T'HH:mm:ss";
+    private final static String dateFormat = "yyyy-MM-dd'T'HH:mm:ss";
+    private final static String humanReadableFormat = "dd MMM yyyy HH:mm:ss";//Mon, 13 Apr 2015 22:59:26
```

Comment:
Looks like you're working with 2 formats: the first one is used to store data in the database and the second one is used to show it to the user. Try to avoid such naming like `humanReadable` as it doesn't tell a developer your intention but just saying that you weren't able to pick up a better name. BTW, they are constants and should be named in a different manner ;) Let's rename the first one to the `DATABASE_FORMAT` and the second one to the `UI_FORMAT`. Corresponding methods would be `formatForDatabase()` and `formatForUi()`.